specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2005,6 +2116,20 @@
             "modelAsString": true
           }
         },
+        "enableOSPatching": {
+          "type": "boolean",
+          "default": false,
+          "title": "Enable Auto OS Patching.",
+          "description": "Enable Auto OS Patching. Possible values are: true, false.",
+          "x-nullable": true
+        },
+        "releaseQuotaOnStop": {
+          "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

Consider using string enums
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md)

Reply:
It will only have 2 values, other boolean type properties in this file are not enums, I'd prefer to keep it as boolean